proto/elys/incentive/incentive.proto

```diff
@@ -11,10 +11,10 @@ message IncentiveInfo {
 	string amount = 1 
         [ (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"amount\""];
     // epoch identifier
-	string epochIdentifier = 2;
+	string epoch_identifier = 2;
     // start_time of the distribution
     google.protobuf.Timestamp start_time = 3 
         [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"start_time\""];
     // distribution duration
-	int64 numEpochs = 4;
+	int64 num_epochs = 4;
```

Comment:
@kenta-elys We should add a current_epoch so we can track where we are in the period easier.

Reply:
You can look at the x/commitment vesting functions to see how it's used there

Reply:
Yep, got it.

Reply:
Done, fixed.